### PR TITLE
[Update] Screenshots

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Addresses partials the format updates that need to be made, moving the images to S3 and updated the links

# Links

Fixes https://github.com/timescale/docs-private/issues/126

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Comment by charislam** [comment]
Thanks Rajie!

Two things:
1. Can you explicitly add a width and height to the images? As in `` (replace with the actual width and height) This helps avoid layout shift after the image loads
2. This is a minor nit, but we actually have a cleaner URL for the assets bucket, you can directly use `https://assets.timescale.com` rather than the one with the Amazon s3 domain in front

File: _partials/_graphing-ohlcv-data.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ up to connect to your TimescaleDB database.
     graph to the dashboard.
 
     <img class="main-content__illustration"
-        src="https://s3.amazonaws.com/assets.timescale.com/docs/images/Grafana_candlestick_1day.png"
+        src="https://s3.amazonaws.com/assets.timescale.com/docs/images/Grafana_candlestick_1day.webp"
```
Comment (charislam) [inline _partials/_graphing-ohlcv-data.md:25]: test

Comment (charislam) [inline _partials/_graphing-ohlcv-data.md:25]:
```suggestion
        src="https://assets.timescale.com/docs/images/Grafana_candlestick_1day.webp"
```

**Comment by Loquacity** [comment]
Nice work, Rajie! 💖 What tool are you using for the image conversion to webp?

**Pull request merged**